pkg/corerp/api/v20231001preview/volume_conversion_test.go

```diff
@@ -87,7 +87,7 @@ func TestVolumeConvertFromValidation(t *testing.T) {
 	}
 
 	for _, tc := range validationTests {
-		versioned := &HTTPRouteResource{}
+		versioned := &VolumeResource{}
```

Comment:
How did this work before?

Reply:
I think the test had to change since it was using HTTPRoutes before but now we removes HTTPRoutes from code base.